spec/requests/well_known/node_info_spec.rb

```diff
@@ -19,6 +19,10 @@
             include(
               rel: 'http://nodeinfo.diaspora.software/ns/schema/2.0',
               href: include('nodeinfo/2.0')
```

Comment:
It may make sense to make this `end_with` too, instead of `include`, since this is a routing generated URL, and we expect it to be at the end of the URL, not anywhere within it.